addons/l10n_es_pos/models/pos_config.py

```diff
@@ -19,7 +19,7 @@ def _default_sinv_journal_id(self):
     )
     l10n_es_simplified_invoice_journal_id = fields.Many2one(
         comodel_name='account.journal',
-        domain="[('type', '=', 'sale')]",
+        domain=[('type', '=', 'sale')],
```

Comment:
it should be a lambda here ?

Reply:
it doesn't need to. I don't need to access something from `self`.